src/Tools/Generator.php

```diff
@@ -76,6 +76,31 @@ public function processRoute(Route $route, array $rulesToApply = [])
         return $parsedRoute;
     }
 
+    protected function getBodyParameters(ReflectionMethod $method, array $tags)
+    {
+        foreach ($method->getParameters() as $param) {
+            $paramType = $param->getType();
+            if ($paramType === null) {
+                continue;
+            }
+
+            $formRequestClassName = version_compare(phpversion(), '7.1.0', '<')
```

Comment:
Good work. One last thing: let's rename this to `$parameterClassName`, since we don't know at this point if it is a FormRequest. You can just accept my suggestions below.

```suggestion
            $parameterClassName = version_compare(phpversion(), '7.1.0', '<')
```